arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,1127 @@
+                        "description": "Service unavailable"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.PowerBI/workspaceCollections": {
+            "get": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "getWorkspacesCollectionsInResourceGroup",
+                "description": "The **Get Power BI Workspace Collections in Resource Group** operation retrieves all existing Power BI Workspace Collections in the specified resource group.",
+                "consumes": [
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "name": "subscriptionId",
+                        "in": "path",
+                        "description": "Azure subscription id",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "resourceGroupName",
+                        "in": "path",
+                        "description": "Azure resource group",
+                        "required": true,
+                        "type": "string"
+                    },
+                    {
+                        "name": "api-version",
+                        "in": "query",
+                        "description": "Azure API version",
+                        "required": true,
+                        "type": "string"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Get workspaces response",
+                        "schema": {
+                            "$ref": "#/definitions/WorkspaceCollectionList"
+                        }
+                    },
+                    "400": {
+                        "description": "Request is not well formed"
+                    },
+                    "403": {
+                        "description": "Authentication failed"
+                    },
+                    "404": {
+                        "description": "Resource group does not exist"
+                    },
+                    "408": {
+                        "description": "Operation could not be completed within the permitted time"
+                    },
+                    "503": {
+                        "description": "Service unavailable"
+                    }
+                }
+            }
+        },
+        "/subscriptions/{subscriptionId}/providers/Microsoft.PowerBI/workspaceCollections": {
+            "get": {
+                "tags": [
+                    "WorkspaceCollections"
+                ],
+                "operationId": "getWorkspacesCollectionsInSubscription",
```

Comment:
Typo again? Remove the double plural of workspaces and collections.

`getWorkspacesCollectionsInSubscription`
`getWorkspaceCollectionsInSubscription`